### ISSUE TITLE
Loading stops at [Chisel] Init Finished

### ISSUE THREAD

**Issue by mrbaggins** [Beyond-Reality/Beyond-Reality-Divergence#53]
For some reason, 3/4 times I try to start this instance in ATLauncher, loading will just stop at [Chisel] Init finished. The Minecraft screen says it's trying to initialise ComputerCraft. It will not progress if it stops here, and CPU usage drops to nearly 0% (from 50-70% the rest of the time). It occasionally will stop at other points (Has happened twice) but by far the Chisel spot happens more than anything else, at least twice for every time I successfully load the pack.

It is as the 1568MB mark for RAM usage. I've set ATLauncher to assign 3GB at startup and 4GB Maximum (of my 8GB). Using 64bit Java 8. Playing on 1.2 version.

**Comment by Dominance** [comment]
Never change the amount allocated on start up. Switch it back to 256mb Make sure that your permgen is 256 - 512

**Comment by mrbaggins** [comment]
righto, I'll do that, but I only changed it originally because of the same problem. I can't recall if it was the Chisel line or not, but it locked up, so I upped the memory, and it worked straight away. It's been sporadic since. I'll do that this evening and see if that helps.

**Comment by Dominance** [comment]
Let me know how it works out. If its still giving you issues toss me your fml-latest log within the root of your instance folder / logs - > File < reopen the issue and pop it in thanks!

**Comment by mrbaggins** [comment]
No good. I changed back, ran it to test, worked fine. Updated to 1.2.1, stopped at the same spot.

Console log: http://paste.atlauncher.com/view/1612cc1d
fml-latest: http://pastebin.com/JYtFfpKM

Honestly, it nearly seems to be linked to whether or not I've taken screen focus away from the game. If I hit Play, then browse the net, it fails 90% of the time. If I hit play and walk away, it works better than 50:50.
